CHANGELOG.md

```diff
@@ -24,6 +24,12 @@ Fixed - for any bug fixes
 Security - in case of vulnerabilities
 -->
 
+## [1.5.] - 2023-11-02
```

Comment:
Missing patch version, which is OK since you will need 1.5.3

Reply:
Fixed